impl_core/src/main/java/io/opencensus/implcore/tags/propagation/SerializationUtils.java

```diff
@@ -76,17 +77,19 @@ private SerializationUtils() {}
     // Use a ByteArrayDataOutput to avoid needing to handle IOExceptions.
     final ByteArrayDataOutput byteArrayDataOutput = ByteStreams.newDataOutput();
     byteArrayDataOutput.write(VERSION_ID);
+    int totalChars = 0;
```

Comment:
maybe add a note that chars == bytes because we know them to be ascii

Reply:
Sure, added.